src/app/client/src/app/modules/program-dashboard/components/program-datasets/program-datasets.component.ts

```diff
@@ -104,7 +112,31 @@ export class DatasetsComponent implements OnInit, OnDestroy {
   displayFilters:any = {};
   loadash = _;
   pdFilters:ConfigFilter[] = [];
-  configuredFilters:any = {}
+  configuredFilters:any = {};
+  appliedFilters:object = {};
+  blocks:object[] = [
+    {
```

Comment:
why is there a data hardcoded
and why in sunbird this code is there?

Reply:
Apologies, had put in for testing, will remove this, thanks.